packages/client/modules/meeting/components/SelectScaleDropdown.tsx

```diff
@@ -43,21 +43,21 @@ const StyledMenu = styled(Menu)({
 
 const SelectScaleDropdown = (props: Props) => {
   const {menuProps, dimension} = props
-  const {team} = dimension
-  const {scales} = team
-  const defaultActiveIdx = useMemo(() => scales.findIndex(({id}) => id === dimension.selectedScale.id), [dimension])
+  const {closePortal} = menuProps
+  const {selectedScale, team} = dimension
+  const {id: seletedScaleId} = selectedScale
+  const {id: teamId, scales} = team
+  const defaultActiveIdx = useMemo(() => scales.findIndex(({id}) => id === seletedScaleId), [dimension])
 
   const atmosphere = useAtmosphere()
   const {onError, onCompleted, submitting, submitMutation} = useMutationProps()
 
   const addScale = () => {
-    const {menuProps} = props
-    const {closePortal} = menuProps
```

Comment:
nit: always try to do all the destructuring at the top, that way you do it once & know that all handlers are referencing the same thing. this will help with bugs as you write more things with `useRef` where the object contents is mutable